src/components/list/list.tsx

```diff
@@ -70,12 +146,120 @@ export class List implements InteractiveComponent {
   // --------------------------------------------------------------------------
 
   render(): VNode {
+    const { loading, label } = this;
     return (
-      <Host role="list">
-        <div class={CSS.container}>
-          <slot />
-        </div>
-      </Host>
+      <div class={CSS.container}>
+        {loading ? <calcite-scrim class={CSS.scrim} loading={loading} /> : null}
+        <table
+          aria-busy={toAriaBoolean(loading)}
+          aria-label={label || ""}
+          class={CSS.table}
+          onKeyDown={this.handleListKeydown}
+          role="treegrid"
+        >
+          <tbody class={CSS.tableContainer}>
+            <slot onSlotchange={this.handleDefaultSlotChange} />
+          </tbody>
+        </table>
+      </div>
     );
   }
+
+  // --------------------------------------------------------------------------
+  //
+  //  Private Methods
+  //
+  // --------------------------------------------------------------------------
+
+  handleDefaultSlotChange = (event: Event): void => {
+    updateListItemChildren(getListItemChildren(event));
+  };
+
+  setActiveListItem = (): void => {
+    const { listItems } = this;
+
+    if (!listItems.some((item) => item.active)) {
+      if (listItems[0]) {
+        listItems[0].active = true;
+      }
+    }
+  };
+
+  updateSelectedItems = debounce((): void => {
```

Comment:
Can this be made private to avoid the type issue we encountered recently? This applies to other debounced methods.